src/features/courses/course-route.ts

```diff
@@ -445,7 +444,38 @@ router.put('/topic/extend',
             next(httpResponse.Ok('Extended topic successfully', updatesResult));
         }));
 
-router.put('/topic/:id',
+router.put('/topic/:id/regrade',
+    authenticationMiddleware,
+    validate(regradeCourseTopicValidation),
+    paidMiddleware('Regrading topic'),
+    // This is due to a typescript issue where the type mismatches extractMap
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Unnecessary log

Reply:
Not unnecessary, if we need to break a linting rule we should disable it for the line and explain it

Reply:
Oh I misread, yeah I'll remove this